public/app/features/query/components/QueryGroupOptions.tsx

```diff
@@ -413,7 +413,7 @@ function getStyles(theme: GrafanaTheme2) {
       justifySelf: 'left',
     }),
     operator: css({
-      color: theme.v1.palette.orange,
+      color: 'rgb(0, 155, 101)',
```

Comment:
This gives the `=` operator visual in the query group options editor the NI green color.